frontend/src/core/hotkeys/hotkeys.ts

```diff
@@ -81,6 +81,11 @@ const DEFAULT_HOT_KEY = {
     group: "Creation and Ordering",
     key: "Mod-Shift-2",
   },
+  "cell.addColumnBreakpoint": {
+    name: "Add column breakpoint",
+    group: "Creation and Ordering",
+    key: "Mod-Shift-3",
```

Comment:
What should the hotkey for this be? Cmd+Shift+3 is already taken for screenshots on MacOS

Reply:
`Mod+Shift+/`?